packages/fxa-auth-server/lib/senders/emails/partials/automatedEmailChangePassword/index.stories.ts

```diff
@@ -0,0 +1,23 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { Meta } from '@storybook/html';
+import { storyWithProps } from '../../storybook-email';
+
+export default {
+  title: 'Partials/automatedEmail/automatedEmailChangePassword',
```

Comment:
Can we put this under `Partials/footers/*`? I think it'll make it a lot more obvious for Product etc. when they're looking at storybook, that this is going to show our footers.

Reply:
Sure thing.

I think @vbudhram was asking if he should extract _more_ content of these inactive account emails into partials (vs questioning if we should add storybooks for the partials :))